### DISCUSSION TITLE
How to contribute to node js website?

### DISCUSSION THREAD

**Discussion by yaten2302** [nodejs/nodejs.org#6251] [Q&A] [answered]
Hi, I want to contribute in the redesigning of the nodejs website. Could you please tell that is this project restricted only to the members of nodejs community or anyone can contribute?

If it's open to everyone, could you please give some tips so that I can contribute to this project.

Thanks

**Comment by AugustinMauroy** [comment, marked as answer]
YEEAAAHH the project is open for everyone !

if you want to contribute there some [issues with good for first issue label](https://github.com/nodejs/nodejs.org/issues?q=is%3Aissue+is%3Aopen+label%3A%22good+first+issue%22).

If you want to understand how the project is made, here are some files that may help:
- [Node.js Collaborator Guide](https://github.com/nodejs/nodejs.org/blob/main/COLLABORATOR_GUIDE.md)
- [Node.js Website Contributing Guide](https://github.com/nodejs/nodejs.org/blob/main/CONTRIBUTING.md)

**Reply by yaten2302** [reply]
Really sorry for the inconvenience, I wanted to ping someone in the org, by-mistake pinged him 🙏

Thank you so much for your guidance, I will surely try to contribute to the website. Also, could you please guide me that for contributing do we have to learn typescript?

I know HTML, CSS, Tailwind CSS and currently learning JavaScript.

**Reply by AugustinMauroy** [reply]
Any contribution is welcome, but it's true that if you're not comfortable with react and nextjs the infra will seem complicated. You can also help with content.

**Reply by ovflowd** [reply]
FYI the README has everything you need to get started with the repository, and this discussion has the context of the redesign https://github.com/nodejs/nodejs.org/discussions/5131, you can also check the current open issues for work that can be done :)

**Reply by yaten2302** [reply]
Thank you so much for your suggestions👍
I'll try to learn react and nextjs ASAP to contribute to this project. Also, could you please guide that how can we contribute to the content?
Also, could you please tell that is there any discord or slack channel for `nodejs` where we can get guidance regarding how to start with the tech stack of this website?

**Reply by AugustinMauroy** [reply]
takes time to learn. What I can suggest is that you make a blog/portfolio with nextjs and the content in markdown.
Mais la discussion devient hors sujet 😄.

For the slack there are a `nodejs-website` channel.
[openjsf slack invite link](https://slack-invite.openjsf.org/)

But keep in mind that the website team isn't here to give lessons on nextjs. If you want some help, you can contact me personally.


for helping with content you have some [issues with label "content"](https://github.com/nodejs/nodejs.org/issues?q=is%3Aissue+is%3Aopen+label%3Acontent). they are very well explained

**Reply by yaten2302** [reply]
Thanks for the slack invite link.
Yes, I completely understand that this channel is only for discussing about the nodejs.org repository and not for teaching👍
May I contact you on LinkedIn if it's ok with you?

**Reply by AugustinMauroy** [reply]
If you want to contact me look up on my [website](augustinmauroy.github.io) there are an section "find me".

I'm going to close this discussion because if I don't we're going to fall into off-topics.
